content/en/_index.md

```diff
@@ -33,7 +33,7 @@ Learn more
 - [C++]({{< relref "/docs/languages/cpp/quickstart" >}})
 - [Java]({{< relref "/docs/languages/java/quickstart" >}})
 - [Python]({{< relref "/docs/languages/python/quickstart" >}})
-- [C#]({{< relref "/docs/languages/csharp/quickstart" >}})
+- [C#]({{< relref "/docs/languages/csharp" >}})
```

Comment:
I'd suggest dropping C# from the list. WDYT?

Reply:
In fact, we might want to change the order a bit. The new order based on page access in the past 6 months, from most popular to least is:

- Go
- Python
- C++
- Java

Reply:
C# is still perfectly valid language to use in gRPC, so I don't think we should drop it. Reordering languages can be done as a followup.

Reply:
Actually, you're right. I didn't notice the "..." ellipsis before. I removed C# from the quick selection.